tests/generators/bls/main.py

```diff
@@ -20,6 +20,7 @@ def hash(x):
 
 F2Q_COEFF_LEN = 48
 G2_COMPRESSED_Z_LEN = 48
+DST = b'BLS_SIG_BLS12381G2-SHA256-SSWU-RO_POP_'
```

Comment:
For consistency with the spec, this ought to be `BLS_SIG_BLS12381G2-SHA256-SSWU-RO-_POP_` (note the extra `-` before `_POP_`).